.eslintrc.js

```diff
@@ -29,24 +29,10 @@ module.exports = {
     '@typescript-eslint/ban-ts-comment': ['warn'],
     // TODO: change no-console to error out for prod
     'no-console': ['warn'],
-    // TODO: fix copied js code that makes this warn necessary
-    'no-use-before-define': ['warn'],
-    'linebreak-style': 0,
-    //TODO: make requests asynchronous
```

Comment:
todo is resolved

Reply:
What requests did this TODO refer to? Or was it more of a general todo to make all requests async?

Reply:
we used to do something like

```js
for (...) {
  await ...
}
```

This is bad because you are not making things run in parallel. Now we no longer have this kind of code.